setup.py

```diff
@@ -58,7 +58,7 @@ def get_long_description():
       license='Apache 2.0',
       packages=['tools'],
       install_requires=['boolean.py', 'jupyter', 'matplotlib>=2.1', 'numpy>=1.17', 'projectq>=0.4', 'qiskit>=0.20.0',
-                        'nbimporter', 'quantuminspire'],
+                        'nbimporter', 'quantuminspire', 'rich'],
```

Comment:
The usage is too little to add this package by default. I opt for leaving it out.

Reply:
I agree. Removed.